azure-functions/acbs-function/api.js

```diff
@@ -154,7 +205,7 @@ const put = async (endpoint, payload, etag) => {
 /**
  * Invokes TFS PATCH endpoint
  * @param {string} endpoint TFS endpoint
- * @param {object} payload Payload object
+ * @param {Object} payload Payload object
```

Comment:
I think these should be lowercase - outside of this PR:

@param {Object} - 4 instances
@param {object} - 385 instances

Reply:
I actually just looked this up due to the array question -- and I think we've made a mistake here, where it should be `Object` technically as objects arent primatives. But I think that's another PR thing